projects/origin-release/pipeline.yaml

```diff
@@ -31,103 +31,548 @@ parameters:
   description: Git source context base to use for building the Origin release environment image
   value: projects/origin-release
   required: false
+- name: PUBLISH_REPO
```

Comment:
I added this to allow for testing with an org other than openshift. I've published images built with this PR to https://hub.docker.com/r/openshiftmultiarch/origin-release/tags/

Reply:
Figure by the time we check this in and let it loose we won't need the param?

Reply:
If we go down that route, then there wouldn't be a way to test the pipeline easily without actually publishing the images to the actual images.  If that isn't a concern, then do we need any of the other params?